packages/cli-platform-android/src/config/getAndroidProject.ts

```diff
@@ -94,9 +94,9 @@ export function parsePackageNameFromAndroidManifestFile(
 
 export function parseNamespaceFromBuildGradleFile(buildGradle: string) {
   // search for namespace = inside the build.gradle file via regex
-  const matchArray = buildGradle.match(/namespace\s*[=]*\s*"(.+?)"/);
+  const matchArray = buildGradle.match(/namespace\s*[=]*\s*("(.+?)"|'(.+?)')/);
```

Comment:
I believe we should rewrite the regex to make sure the namespace is still in `matchArray[1]`

Reply:
I've tried several regexes but failed. Could anyone help?

Reply:
This should work `namespace\s*[=]\s*["'](.+?)["']`